### PR TITLE
Add performance to Android Wizard

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

**Review by philipphofmann — APPROVED** [review]
LGTM 🚀

**Review by bruno-garcia — APPROVED** [review]
Thank you

File: src/wizard/android/index.md [hunk 1]
```diff
@@ -69,6 +79,39 @@ class MyActivity : AppCompatActivity() {
...
+You can measure the performance of your code by capturing transactions and spans.
+
+```kotlin
+import io.sentry.Sentry
+
+// Transaction can be started by providing, at minimum, the name and the operation
+val transaction = Sentry.startTransaction("updateRepos", "task")
+
+// Transactions can have child spans (and those spans can have child spans as well)
+val span = transaction.startChild("test-child-operation")
```
Comment (philipphofmann) [inline src/wizard/android/index.md:102]: `l`: Maybe we should choose better operation names, as their are used for filtering.

File: src/wizard/android/index.md [hunk 2]
```diff
@@ -69,6 +79,40 @@ class MyActivity : AppCompatActivity() {
...
```
Comment (philipphofmann) [inline src/wizard/android/index.md:103]: l: Maybe we should choose better operation names, as they are used for filtering.
```suggestion
val transaction = Sentry.startTransaction("Update Repos", "db")

// Transactions can have child spans (and those spans can have child spans as well)
val span = transaction.startChild("db", "Update first repo")
```

**Pull request merged**